frontend/src/lib/k8s/api/v2/webSocket.ts

```diff
@@ -210,6 +213,7 @@ export const WebSocketManager = {
       query,
       userId: userId || '',
       type: 'REQUEST',
+      token: getToken(clusterId),
```

Comment:
I think if there is no token the token field should not be added to the message?
Because the check in goland compares token to nil. 


Something like this (untested):
```ts
const token = getToken(clusterId)
if (token) {
  requestMsg.token = token;
}
```

(Maybe runs fine as is... but this might be a bit better to not include the token field if there's not one for debugging too.)

Reply:
wouldn't go parse undefined or null as nil?

Reply:
JSON.stringify removes fields with undefined values.

So it’s fine as is.